src/types/nutrition.type.ts

```
@@ -23,6 +25,35 @@ export type PurposeValue = {
+export type MeasurementUnitValues = {
+  label: string;
+  name: string;
+  unit: string;
+};
+
+export type MeasurementUnitType = keyof typeof MeasurementUnitEnum;
+
 export type DailyMealCalories = {
   [key in string]: { calories: number; caloriesGoal: number };
 };
+elop
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)